src/config.rs

```diff
@@ -5,8 +5,14 @@ use std::path::PathBuf;
 #[derive(Serialize, Deserialize, Debug, Default, Clone)]
 pub struct Config {
     pub database: Database,
+    pub orbits: OrbitsConfig,
+}
+
+#[derive(Serialize, Deserialize, Debug, Default, Clone)]
+pub struct OrbitsConfig {
+    pub public: bool,
```

Comment:
I would either add a default value or un-comment the setting in the config file.

Reply:
the default of `bool` is `false`, which I think is a reasonable default, should it be made more explicit?